eventhubs/azure-spring-cloud-starter-eventhubs/eventhubs-integration/pom.xml

```diff
@@ -2,59 +2,40 @@
 <project xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
          xmlns="http://maven.apache.org/POM/4.0.0"
          xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
-  <parent>
-    <groupId>org.springframework.boot</groupId>
-    <artifactId>spring-boot-starter-parent</artifactId>
-    <version>2.5.4</version> <!-- {x-version-update;org.springframework.boot:spring-boot-starter-parent;external_dependency} -->
-  </parent>
-  <modelVersion>4.0.0</modelVersion>
+    <parent>
+        <groupId>com.azure.spring</groupId>
+        <artifactId>azure-spring-boot-samples</artifactId>
+        <version>1.0.0</version>
+    </parent>
```

Comment:
add this: <relativePath>../../../pom.xml</relativePath>

Reply:
updated